### ISSUE TITLE
Table component is not keyboard accessible

### ISSUE THREAD

**Issue by kyleoliveiro** [auth0/cosmos#872]
Sortable table header cells and clickable table rows cannot be interacted with via a keyboard, as they do not have the necessary `tabindex` attribute.

**Comment by kyleoliveiro** [comment]
This issue also applies to the following components. Not sure whether to create a separate issue for each, so I'll list them here for now:

**Components lacking keyboard accessbility**
- Alert (dismiss button)
- Tag
- Tooltip
- Dialog (cannot access dialog contents)
- Resource list (only action buttons are accessible)

**Comment by landitus** [comment]
Thanks @kyleoliveiro ! We are planning to add keyboard accessibility to all, but Table seems like a good start.

**Comment by rfbotto** [comment]
I would like to tackle this if possible @landitus! :) Just to make sure i understood, both table header cells and table rows, should have the `tabIndex` set in order for them to be focusable by keyboard, is that right?
